components/core/src/clp_s/JsonConstructor.hpp

```diff
@@ -30,7 +30,7 @@ struct JsonConstructorOption {
     std::string archive_id;
     std::string output_dir;
     bool ordered{false};
-    size_t ordered_chunk_size{0};
+    size_t ordered_chunk_size{};
```

Comment:
I'd prefer to keep the current default behaviour. We can add more to the description of the argument to describe that 0 gives you no chunking.

I think it makes more sense for the package to pick its behaviour inside of the package config instead of forcing it into the CLI.

Reply:
Right, now I agree that it should be in the package config. It is never there though, maybe we should consider changing this behavior in another PR to let the default to be configurable